src/main/java/org/fundacionjala/virtualassistant/controller/TextRequestController.java

```diff
@@ -0,0 +1,23 @@
+package org.fundacionjala.virtualassistant.controller;
+
+import org.fundacionjala.virtualassistant.models.RequestEntity;
+import org.fundacionjala.virtualassistant.service.TextRequestService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/textRequest")
+public class TextRequestController {
+
+    TextRequestService textRequestService;
+
+    @GetMapping("/{userId}?contextId={contextID}")
```

Comment:
here contextId is a request variable not a path variable, please change that accordingly